### ISSUE TITLE
Check the RGBtoDHS model's performance

### ISSUE THREAD

**Issue by liketheflower** [liketheflower/YONOD#4]
- step 1: We first train 50 epochs based on RGB  image with pre-trained weights from COCO
- step 2: Then train 50 epochs based on DHS images with the weight of epoch 50 in step 1.
- Step 3: test the validation set based on DHS images from the epochs from step 1 (The RGB val was already tested in step 1).
- Step 4: test the validation set based on  RGB images from the epochs from step 2 (The DHS was already tested in step 2).
